### PR TITLE
Commune methods

### LINKED ISSUE #75: Commune methods
Commune creation method for the model, similar to 

image

but with the commune fields 

Also a method which checks if the user activating it is the one who created the commune, and if so allows him to add members

### PR DESCRIPTION
**Pull request by RonTuretzky**
Fix #75

### CODE REVIEW

File: commune_app/all_models/communes.py [hunk 1]
```diff
@@ -16,6 +18,7 @@ class Commune(models.Model):
     name = models.CharField(max_length=100, unique=True, blank=False)
     description = models.CharField(max_length=250, blank=True)
     wallet = models.IntegerField(default=0, validators=[validate_wallet])
+    founder = models.ForeignKey(settings.AUTH_USER_MODEL, on_delete=models.CASCADE, default=346)  # 346 = Admin
```
Comment (kobihk) [inline commune_app/all_models/communes.py:21]: Here you added one more class member(DB column) to the Commune model 
but I can't see where you migrate it.

btw. 
do we really need it? 
maybe we can keep up without this field?

**Review by kobihk — CHANGES_REQUESTED** [review]

File: commune_app/all_models/communes.py (hunk 1 above)
Comment (RonTuretzkyMigal) [reply commune_app/all_models/communes.py:21]: I have the migrations on the branch, I will add it 

I added it in order to be able to verify that if someone is removing people from the commune, it is only the person who made the commune. 

Is there a different way of verifying that without this field?

Comment (RonTuretzky) [reply commune_app/all_models/communes.py:21]: I have the migrations on the branch, I will add it

I added it in order to be able to verify that if someone is removing people from the commune, it is only the person who made the commune.

Is there a different way of verifying that without this field?

Comment (kobihk) [reply commune_app/all_models/communes.py:21]: in the built-in auth-user we have the field is_superuser
I thought maybe we can use it for this purpose

**Comment by RonTuretzky** [comment]
@kobihk PR amended to your suggestion

**Review by FaezZangariya — APPROVED** [review]

**Review by NicolePauzner — APPROVED** [review]

**Review by FaezZangariya — APPROVED** [review]

File: commune_app/templates/commune_app/login.html [hunk 2]
```diff
@@ -13,7 +13,7 @@
                 <form method="post">{% csrf_token %}
 
                     <div class="d-flex flex-row align-items-center mb-1">
-                        <input type="email" placeholder="Email address *" class="form-control m-1" name="email" />
+                        <input type="text" placeholder="Username *" class="form-control m-1" name="username" />
```
Comment (kobihk) [inline commune_app/templates/commune_app/login.html:16]: Please look at:
https://github.com/redhat-beyond/commune/blob/main/commune_app/templates/commune_app/login.html#L16
These changes are already in the code.
Could you please rebase this PR on the latest code?

**Review by kobihk — CHANGES_REQUESTED** [review]

File: commune_app/templates/commune_app/login.html (hunk 2 above)
Comment (RonTuretzky) [reply commune_app/templates/commune_app/login.html:16]: I tried rebasing again @kobihk

**Review by EliasAndry — APPROVED** [review]

**Review by FaezZangariya — DISMISSED** [review]

**Review by EliasAndry — DISMISSED** [review]

**Pull request closed without merging**